extension/solarwindsapmsettingsextension/go.mod

```diff
@@ -3,12 +3,15 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/extension/solar
 go 1.21
 
 require (
+	github.com/solarwindscloud/apm-proto v0.0.0-20231107001908-432e697887b6
```

Comment:
If you're able to, could you add semver tags to this repo to help make it easier to maintain and avoid accidentally bumping to a new version that contains breaking changes (either in API or behaviour)?

Reply:
Done. Added semver tags to github.com/solarwindscloud/apm-proto and now it is using `github.com/solarwindscloud/apm-proto v1.0.3`